### ISSUE TITLE
Bug twinning: numbers of columns of arguments do not match

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/pirouette#103]
From [a Travis build log](https://travis-ci.org/richelbilderbeek/pirouette/builds/489932365#L1816):

```
  ── 2. Error: most_evidence, with twinning (@test-pir_run.R#590)  ───────────────
  numbers of columns of arguments do not match
  1: pir_run(phylogeny = phylogeny, pir_params = pir_params) at testthat/test-pir_run.R:590
  2: rbind(df, df_twin)
  3: rbind(deparse.level, ...)
  4: stop("numbers of columns of arguments do not match")
```

Expose the bug:

```
  skip("Issue 103, #103")
```

**Comment by Giappo** [comment]
In test-pir_run there is still a skip("Issue 103, #103").
I need this to be fixed to close #114 
Unfortunately, the experiments that are meant to be set cannot run on windows os. For this reason I assign @richelbilderbeek . I hope it's fine for you.

**Comment by richelbilderbeek** [comment]
Agreed, will work on it :+1:
